### ISSUE TITLE
HiBench doesn't like multiple Yarn ResourceManagers

### ISSUE THREAD

**Issue by C-Newman** [intel-hadoop/HiBench#138]
The file bin/functions/load-config.py looks for the yarn resource manager with this command:
match_address=re.findall("\<property\>\s*\<name\>\s*yarn.resourcemanager.address\s
arn.resourcemanager.address\s*\<\/name\>\s*\<value\>([a-zA-Z\-\._0-9]+)(:\d+)\<\/value\>", f.read())

That's fine with one resource manager.  With two, I see the following in yarn-site.xml:

  <property>
    <name>yarn.resourcemanager.address.rm1963</name>
    <value>Server1.company.com:8032</value>
  </property>
...
  <property>
    <name>yarn.resourcemanager.address.rm1538</name>
    <value>Server2.company.com:8032</value>
  </property>

The HiBench complaint is:
AssertionError: Get workers from yarn web UI page failed, reason:Unknown resourcemanager, please check `hibench.hadoop.configure.dir` and "yarn-site.xml" file
please set `hibench.masters.hostnames` and `hibench.slaves.hostnames` manually

**Comment by lvsoft** [comment]
Yes, HiBench's built-in auto detection mechanism rely on the assumption of a relatively simple environment. Basically auto detection is designed to cover 90% simple cases for starters.

For complex cases, you can bypass auto detection by override the value failed to detect, and improvements for more complex environment detection are always welcome.
